pkg/server/plugin/datastore/sqlite/sqlite_test.go

```diff
@@ -461,7 +461,38 @@ func Test_FetchRegistrationEntries(t *testing.T) {
 }
 
 func Test_UpdateRegistrationEntry(t *testing.T) {
-	t.Skipf("TODO")
+	ds := createDefault(t)
+
+	entry1 := &common.RegistrationEntry{
+		Selectors: []*common.Selector{
+			{Type: "Type1", Value: "Value1"},
+			{Type: "Type2", Value: "Value2"},
+			{Type: "Type3", Value: "Value3"},
+		},
+		SpiffeId: "spiffe://example.org/foo",
+		ParentId: "spiffe://example.org/bar",
+		Ttl:      1,
+	}
+
+	createRegistrationEntryResponse, err := ds.CreateRegistrationEntry(&datastore.CreateRegistrationEntryRequest{entry1})
+	require.NoError(t, err)
+	require.NotNil(t, createRegistrationEntryResponse)
+
+	entry1.Ttl = 2
+	updReq := &datastore.UpdateRegistrationEntryRequest{
+		RegisteredEntryId: createRegistrationEntryResponse.RegisteredEntryId,
```

Comment:
Yea, we probably don't need this anymore... it comes from a time where entry id wasn't on registration message, so protobuf message takes an entry id explicitly.

I am thinking I might take a shot at datastore refactor sometime soon... in the meantime, you would prefer to ignore this field? If so, I can update datastore logic to read entry id directly from entry instead of this field, just let me know.

Reply:
I think adding some TODOs with a reference to some issue (the refactoring issue?) to track this would be enough for now.